waku/v2/node/wakunode2_setup_metrics.nim

```diff
@@ -73,4 +68,9 @@ proc startMetricsLog*() =
     discard setTimer(Moment.fromNow(30.seconds), logMetrics)
   
   discard setTimer(Moment.fromNow(30.seconds), logMetrics)
+
+  # Start protocol specific metrics logging
+  when defined(rln) or defined(rlnzerokit):
+    let logRlnMetrics = getRlnMetricsLogger()
+    logRlnMetrics(nil)
```

Comment:
Any reason not to move this into the metrics logging block above and removing the need for adding a separate timed task for RLN metrics?

Reply:
Addressed in [d18a025](https://github.com/status-im/nwaku/pull/1262/commits/d18a0251a1fadb17d51671f8302a3316989f503a)